styles/globals.css

```diff
@@ -26,10 +26,20 @@ body {
     rgb(var(--background-start-rgb));
 }
 
+progress[value]::-webkit-progress-bar {
+  @apply bg-abeg-neutral-80 rounded-[4px];
+}
+progress[value]::-webkit-progress-value {
+  @apply bg-abeg-green rounded-[4px];
+}
+progress[value]::-moz-progress-bar {
+  @apply bg-abeg-green rounded-[4px] h-full;
+}
```

Comment:
It's would be great if we can stick to utility classes only.

We should only extend to `global.css` in the most minimal cases.

This was the case I guess at first, but you should checkout this discussion, it can help achieve this 😉

https://github.com/tailwindlabs/tailwindcss/discussions/3921

Reply:
Thanks for the link